### PR TITLE
[14.0][FIX] account_invoice_consolidated: add migration script

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
:S

### CODE REVIEW

File: account_invoice_consolidated/migrations/14.0.1.0.1/post-migrate.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tools.sql import column_exists
+
+
+def migrate(cr, version):
+    if not column_exists(cr, "account_move", "old_invoice_id"):
```
Comment (flotho) [inline account_invoice_consolidated/migrations/14.0.1.0.1/post-migrate.py:8]: FMI, when does this old_invoice_id supposed to be filled ?

Comment (MiquelRForgeFlow) [reply account_invoice_consolidated/migrations/14.0.1.0.1/post-migrate.py:8]: said in line 9. during account migration, in OU

Comment (flotho) [reply account_invoice_consolidated/migrations/14.0.1.0.1/post-migrate.py:8]: (facepalm) ;-) thanks
I'll learn to better read

**Review by flotho — APPROVED** [review]
LGTM

**Pull request closed without merging**